### PR TITLE
Bug 1217463 - Deal with case where 'places' bookmark is missing, r=se…

### PR DESCRIPTION
**Pull request by danhuang1202**
…anlee

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e8cbf04b7d3d6f8c5fd053afde3f2c5be99f100d

File: tv_apps/browser/js/bookmarkStore.js [hunk 1]
```diff
@@ -80,13 +80,15 @@
...
-            SyncBrowserDB.getBookmark('places', bookmark => {
+            SyncBrowserDB.getBookmark({ parentid: 'places' }, syncBookmark => {
               // make sure if firefox synced data saved in indexdDB
-              if(bookmark) {
+              if(syncBookmark.length > 0) {
                 this.cache.push({
-                  id: bookmark.id,
-                  title: bookmark.title ? bookmark.title : 'Synced Bookmarks',
-                  type: bookmark.type,
+                  id: 'places',
```
Comment (weilonge) [inline tv_apps/browser/js/bookmarkStore.js]: dito

File: tv_apps/browser/js/bookmarkStore.js [hunk 2]
```diff
@@ -80,13 +80,15 @@
             // SyncBrowserDB.getBookmark and BrowserDB.getBookmarks.
             // But now the implementation of these function are not
             // return Promise.
-            SyncBrowserDB.getBookmark('places', bookmark => {
+            SyncBrowserDB.getBookmark({ parentid: 'places' }, syncBookmark => {
```
Comment (weilonge) [inline tv_apps/browser/js/bookmarkStore.js]: Give 'places' a const variable.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=59bbf85880d8aac7388fe44a4aa1e8a994eba362

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=579d877512ae9ee6a3edb238ed3203fb92a11aa0

**Pull request merged**